merqube_client_lib/api_client/merqube_client.py

```diff
@@ -19,6 +22,13 @@ class MerqubeAPIClient(base._IndexAPIClient, base._SecAPIClient):  # noqa
     For a client pertaining to a specific existing index, use MerqubeAPIClientSingleIndex
     """
 
+    def create_identifier(self, provider: Provider, identifier_post: IdentifierUUIDPost) -> dict[str, Any]:
+        """
+        Create a new identifier (in MerQubes system - this does NOT talk to the provider)
+        """
+        payload = json.loads(identifier_post.json(exclude_none=True))
```

Comment:
Maybe check if identifier already exists before trying to create it, in case someone creates it while setting up the ticker with BBG.

Reply:
Also catch the "already exists" error. If it already exists it is fine.

Reply:
@ykifle i think we should pick one:
1) if it already exists its fine (meaining, just ignore 409)
2) if it already exists, it probably means youre trying to use someone elses ticker (raise error)

Reply:
if we want to assume its fine, i think we dont need to do a GET but just swallow the 409. Even if we do a GET theres still a race condition if someone else is making it at the same time, so ignoring the 409 will do it. 

ill go with this for now - though im a little scared between 1 and 2

Reply:
I guess the key is does the existing identifier have the same index_name or not. If it has the same index_name as your index then it is probably case 1. If it is a different index_name then this is case 2.

Reply:
@ykifle  ok - got it, sounds good